shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3754,6 +3754,10 @@ void WebContents::SetBackgroundColor(std::optional<SkColor> maybe_color) {
   }
 }
 
+void WebContents::PDFReadyToPrint() {
+  Emit("pdf-ready-to-print");
```

Comment:
Let's prefix this to indicate it's private for now?

```suggestion
  Emit("-pdf-ready-to-print");
```

Reply:
If we want it to be private, should we remove it from `docs/api/web-contents.md`?